### DISCUSSION TITLE
Feature selection

### DISCUSSION THREAD

**Discussion by renja-g** [fastapi/full-stack-fastapi-template#1314] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
[*] - Auth
[ ] - Email
[ ] - Traefik
...
```

### Description

It would be cool when you would be able to select features you want when setting up the project, like:
```
[*] - Auth
[ ] - Email
[ ] - Traefik
...
```
Because I think most people will not need everything the project has to offer, and it might be unnecessarily complicated for them to manually remove everything they don't need

### Operating System

Other

### Operating System Details

All

### Python Version

3.12

**Comment by YuriiMotov** [comment]
But when the docs will become more complicated as we will need to cover all these use cases..
I think this is out of scope for this project.

This project shows one common way to create and deploy app. It's impossible to cover all users needs
